.github/workflows/e2e-tests.yml

```diff
@@ -232,9 +236,9 @@ jobs:
 
   visual-regression-tests:
     runs-on: ubuntu-20.04
-    timeout-minutes: 30
-    needs: build
-    name: percy-tests-sanity-check-${{ matrix.edition }}
+    timeout-minutes: 45
```

Comment:
It was set to 35 minutes on `master`.

Reply:
I've seen a lot of timeouts for this job in PRs.
It definitely needs more than 30 minutes.